external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -37,6 +99,13 @@ describe('/party-db', () => {
 
       expect(status).toEqual(HttpStatusCode.Ok);
     });
+
+    it(`calls the correct url`, async () => {
+      await get(`/party-db/${VALID}`);
```

Comment:
Slightly confused, looking at `party-db.controller.ts` I can see the function only calls one endpoint `url: '${APIM_MDM_URL}customers',`, where is the following call made when customer does not exist.

Reply:
Ah yep good spot, this was from a previous implementation I'll remove these bits